design/ceph-csi-driver.md

```diff
@@ -8,21 +8,21 @@ common storage features such as create/delete volumes, publish/unpublish volumes
 Kubernetes started to CSI driver alpha support in [1.9](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/), beta support in [1.10](https://kubernetes.io/blog/2018/04/10/container-storage-interface-beta/).
 
 It is projected that CSI will be the only supported persistent storage driver
-in the near feature. In-tree drivers such as Ceph RBD and Ceph FS will be replaced with their respective CSI drivers.
+in the near feature. In-tree drivers such as Ceph RBD and CephFS will be replaced with their respective CSI drivers.
 
 ## Ceph CSI Drivers Status
 
-There have been active Ceph CSI drivers development since Kubernetes 1.9. 
-Both Ceph RBD and Ceph FS drivers can be found at [ceph/ceph-csi](https://github.com/ceph/ceph-csi). Currently the drivers are up to CSI v0.3.0 spec.
+There have been active Ceph CSI drivers developments since Kubernetes 1.9. 
+Both Ceph RBD and CephFS drivers can be found at [ceph/ceph-csi](https://github.com/ceph/ceph-csi). Currently the drivers are up to CSI v0.3.0 spec.
 
 * RBD driver. Currently, rbd driver supports krbd. There is a consideration to support other forms of drivers such as rbd-nbd or TCMU based drivers.
-* CephFS driver. Both Kernel Ceph FS and Ceph FUSE are supported. When `ceph-fuse` is installed, it is chosen as default mounter.
+* CephFS driver. Both Kernel CephFS and Ceph FUSE are supported. When `ceph-fuse` is installed, it is chosen as default mounter.
```

Comment:
Where must `ceph-fuse` be installed? Inside the Ceph CSI container, the host or kubelet (container)?

Reply:
in the csi driver container.